packages/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -214,7 +238,15 @@ export const Tooltip: React.FunctionComponent<TooltipProps> = ({
     top: styles.modifiers.top,
     bottom: styles.modifiers.bottom,
     left: styles.modifiers.left,
-    right: styles.modifiers.right
+    right: styles.modifiers.right,
+    'top-start': styles.modifiers.topLeft,
```

Comment:
Curious why you went with `top-start` instead of `top-left`. Same with the rest that have `start` and `end`.

Reply:
this is the way that it is used within popper which this is based on

Reply:
I see. I wonder if we can leverage those Popper types more for this then, since I see `ComputedPlacement` from `Popper` is close to, if not the same as what we want to use for Tooltip.